docs/_posts/2017-12-15-jekyll-3-7-0-released.markdown

```diff
@@ -0,0 +1,31 @@
+---
+title: "Jekyll 3.7.0 Released"
+description: "Jekyll 3.7.0 brings livereload, ."
+date: 2017-12-15 12:31:40 +0100
+author: dirtyf
+version: 3.7.0
+categories: [release]
+---
+
+We're happy to announce that Jekyll latest minor release is out before Christmas!
+
+Here are a few of the latest gifts 🎁 from our contributors:
+
+ * LiveReload is available as an option during development: `bundle exec jekyll server --livereload`, no more manual page refresh. A big thanks to @awood for this feature and to @andreyvit, livereload author.
+ * New `collection_dir` configuration allows you to store all your collections in a single folder. Your source root folder should now look cleaner.
+ * If you're using a gem-based theme and the `incremental` option, you'll notice some significant speed during the regeneration process, e.g. build time went down from 12s to 2s with @mmistakes [minimal-mistakes theme](https://github.com/mmistakes/minimal-mistakes).
+ * Jekyll will now check to determine whether host machine has internet connection.
+ * A new latin mode is available to better handle URLs slugs.
+ * And of course many bug fixes and updates to our documentation…
+
+Oh, one last thing…
+
+### Please upgrade your Ruby 💎
+
+Prepare for the next major update, as next major version Jekyll 4.0 will drop support for at least Ruby 2.1 and 2.2.
+
+> Ruby 2.2 is now under the state of the security maintenance phase, until the end of the March of 2018. After the date, maintenance of Ruby 2.2 will be ended. We recommend you start planning migration to newer versions of Ruby, such as 2.4 or 2.3. — [Ruby Core Team](https://www.ruby-lang.org/en/news/2017/12/14/ruby-2-2-9-released/)
+
+We strongly encourage you to upgrade to at least Ruby 2.4 or even go with Ruby 2.5 that should be out in the coming days if you want to benefits from the latest language additions.
```

Comment:
recommend at least Ruby 2.3 which won't be going EOL until March '19..

Reply:
Recommend != Require.

We are taking advantage of new language features whenever they can be use in a way that degrades gracefully; I think it is a Good Thing to _recommend_ 2.4 :+1: